docker/fastapi/Pipfile

```diff
@@ -10,6 +10,7 @@ fastapi = "*"
 uvicorn = "*"
 fastapi_utils = "*"
 httpx = "*"
+pydantic = ">=1.0.0,<2.0.0" # prevent unintended upgrade to v2
```

Comment:
you also need to lock this in the dependabot config otherwise dependabot will bump this i believe